README.md

```diff
@@ -41,6 +43,34 @@ sudo ln -s ~/.mason/mason /usr/local/bin/mason
 
 The second line is optional.
 
+#### Submodule
+
+Mason can be added a submodule to your repository instead of creating a global symlink. This is helpful for other contributors to get set up quickly. Make sure to include the final part of the following command `.mason/` so your submodule path has the leading `.` instead of just being `mason/`.
+
+```bash
+git submodule add git@github.com:mapbox/mason.git .mason/
+```
+        
+This will append a few lines to your `.gitmodules` file. Something like this:
+
+```
+[submodule ".mason"]
+    path = .mason
+    url = git@github.com:mapbox/mason.git
```

Comment:
Needs to be http like https://github.com/mapbox/mapbox-gl-native/blob/master/.gitmodules#L3 since travis machines don't have ssh+git user/setup